src/default-markup.ts

```diff
@@ -0,0 +1,235 @@
+          title="${alt}"
+          alt="${alt}"
+          style="${styles.fluid.svgPlaceholder}"
+        >
+
+        ${comment('load the image sources.')}
+        <picture classname="gria-image-sources">
+          <source srcset="${props.srcSet}" sizes="${props.sizes}">
+          <img
+            src="${src}"
+            srcset="${props.srcSet}"
+            sizes="${props.sizes}"
+            title="${alt}"
+            alt="${alt}"
+            loading="lazy"
+            style="${styles.fluid.imageTag}"
+          >
+        </picture>
+
+        ${comment('(inefficiently) add noscript support.')}
+        <noscript>
+          <picture classname="gria-image-sources">
+            <source srcset="${props.srcSet}" sizes="${props.sizes}">
+            <img
+              src="${src}"
+              srcset="${props.srcSet}"
+              sizes="${props.sizes}"
+              title="${alt}"
+              alt="${alt}"
+              loading="lazy"
+              style="${styles.fluid.imageTag}"
+            >
+          </picture>
+        </noscript>
+      </div>
+    `
+  }
+
+  if (props.sharpMethod === 'fixed') {
+    return `
+      <div class="gria-image-wrapper" style="${styles.fixed.imageWrapper}">
+
+        ${comment('show a solid background color.')}
+        <div
+          class="gria-solid-placeholder"
+          title="${alt}"
+          style="${styles.fixed.solidPlaceholder}">
+        </div>
+
+        ${comment('show the blurry base64 image.')}
+        <img
+          class="gria-base64-placeholder"
+          src="${props.base64}"
+          title="${alt}"
+          alt="${alt}"
+          style="${styles.fixed.imagePlaceholder}"
+        >
+
+        ${comment('show a traced SVG image.')}
+        <img
+          class="gria-tracedSVG-placeholder"
+          src="${props.tracedSVG}"
+          title="${alt}"
+          alt="${alt}"
+          style="${styles.fixed.svgPlaceholder}"
+        >
+
+        ${comment('load the image sources.')}
+        <picture classname="gria-image-sources">
+          <source srcset="${props.srcSet}">
+          <img
+            src="${src}"
+            srcset="${props.srcSet}"
+            title="${alt}"
+            alt="${alt}"
+            loading="lazy"
+            style="${styles.fixed.imageTag}"
+          >
+        </picture>
+
+        ${comment('(inefficiently) add noscript support.')}
+        <noscript>
+          <picture classname="gria-image-sources">
+            <source srcset="${props.srcSet}">
+            <img
+              src="${src}"
+              srcset="${props.srcSet}"
+              title="${alt}"
+              alt="${alt}"
+              loading="lazy"
+              style="${styles.fixed.imageTag}"
+            >
+          </picture>
+        </noscript>
+      </div>
+    `
+  }
+
+  // TODO: markup for the resize sharpMethods (?), placeholder below:
+  if (props.sharpMethod === 'resize') {
+    return `<img src="${src}" alt="${alt}">`
+  }
+}
```

Comment:
@fartinmartin realistically `sharpMethod` will only ever be either `fluid`, `fixed` or `resize`, however it's a good practice to guard against all cases. We should add a `return ''` at the end of this function, or better yet throw an error if `sharpMethod` is not one of those 3 states.